src/test/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurerTests.java

```diff
@@ -241,6 +241,18 @@ public void testIsStringAlreadyPresentInFile_isPresent() throws IOException {
         assertThat(isKeyPresentInYMLFile(installer.OPENSEARCH_CONF_FILE, str2), is(equalTo(false)));
     }
 
+    @Test
+    public void testAssumeYesDoesNotInitializeClusterMode() throws IOException {
+        String str1 = "node.name"; // cluster_mode
```

Comment:
nit: These variable names could be more related to their values or placed inline in the `isKeyPresentInYMLFile` call.

Reply:
Applied suggestion in: [3b5d54e](https://github.com/opensearch-project/security/pull/3935/commits/3b5d54ea76658e84ca9214fa5212c61c7d763941)